packages/app/prisma-utils/vite.config.ts

```diff
@@ -6,6 +6,6 @@ import packageJson from './package.json'
 
 /* eslint-disable import/no-default-export */
```

Comment:
We need to replace `eslint` suppression comments with Biome ones (or none if Biome doesn't complain)